images/cellranger/README.md

```diff
@@ -4,7 +4,9 @@ This folder contains a Dockerfile for the cellranger analysis.
 ## Building the image
 
 In order to build this image, the cellranger software and bcl2fastq source code components must be downloaded separately to comply with licensing, and should be placed in this folder (`images/cellranger`).
-- The current version of cellranger is `cellranger-9.0.1.tar.gz` and can be downloaded from [10X Genomics Website](https://10x.vercel.app/support/software/cell-ranger/downloads#download-links) after agreeing to their license terms.
+- The current version of cellranger is `cellranger-9.0.1.tar.xz` and can be downloaded from [10X Genomics Website](https://10x.vercel.app/support/software/cell-ranger/downloads#download-links) after agreeing to their license terms.
+  - Download the `cellranger-9.0.1.tar.xz` file and extract it using the `tar -xJf cellranger-9.0.1.tar.xz` command.
```

Comment:
```suggestion
  - Download the `cellranger-9.0.1.tar.xz` file and extract it using the `tar -xzf cellranger-9.0.1.tar.xz` command.
```
Is this what you meant? I didn't think J was an option?

Reply:
`J` is to for `.xz` archives. But really it could just be `-xf` and tar should autodetect, so that's what I'll do. 

```suggestion
  - Download the `cellranger-9.0.1.tar.xz` file and extract it using the `tar -xf cellranger-9.0.1.tar.xz` command.
```